nodejs/week1/avg.js

```diff
@@ -0,0 +1,16 @@
+/* The "argv" array contains everything given on the command line. 
+The first item (argv[0]) will be the path to node itself, and the second item (argv[1]) will be the path to your script code.
+So a slice starting at 2 will discard both of those and return everything else that was typed on the command line.*/
+
+const arguments = process.argv.slice(2);
```

Comment:
arguments could be a reserved javascript variable so change it to another name
https://www.w3schools.com/js/js_reserved.asp